src/content/quickstart/kind/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2020-03-17T13:36:18+01:00
+title: "KIND"
+---
+
+## KIND
+
+[KIND](https://github.com/kubernetes-sigs/kind) is a tool to run local Kubernetes clusters inside Docker container nodes.
+Submariner provides with scripts that deploy 3 Kubernetes clusters locally, 1 broker and 2 data clusters with submariner
```

Comment:
```suggestion
Submariner provides scripts that deploy 3 Kubernetes clusters locally, 1 broker and 2 data clusters with Submariner
```

Reply:
Further suggestions:  merge L10 here as well.  The broker also has Submariner deployed but w/o a gateway enabled initially - so how about "_..., 1 broker and 2 data clusters with the Submariner dataplane components deployed._"

Reply:
Done